packages/cli/src/ui/components/InputPrompt.tsx

```diff
@@ -48,6 +48,7 @@ export interface InputPromptProps {
   setShellModeActive: (value: boolean) => void;
   onEscapePromptChange?: (showPrompt: boolean) => void;
   vimHandleInput?: (key: Key) => boolean;
+  isShellInputFocused?: boolean;
```

Comment:
rename this field. it isn't anything to to with the shell. it is just whether this input is focused